src/Extsload.sol

```diff
@@ -26,6 +28,7 @@ abstract contract Extsload is IExtsload {
         return value;
     }
 
+    /// @inheritdoc IExtsload
     /// @dev since the function is external and enters a new call context and exits right after execution, Solidity's memory management convention can be disregarded and a direct slice of memory can be returned
```

Comment:
feel like the dev note should go into the natspec in IExtsload

Reply:
will inline it inside the function -- the code comment is specific to the implementation and not the interface imo